db/migrate/20151211163428_add_content_id_to_organisations.rb

```diff
@@ -0,0 +1,13 @@
+class AddContentIdToOrganisations < ActiveRecord::Migration
+  def up
+    add_column :organisations, :content_id, :string
+
+    Organisation.reset_column_information
+
+    ImportOrganisations.new.call
```

Comment:
Ah, didn't know that 3am task existed! We've removed those lines from this migration.